wizard-clock-card.js

```diff
@@ -23,42 +48,23 @@ class WizardClockCard extends HTMLElement {
         }
       }
     }
-    if (this.config.exclude){
-      for (num = 0; num < this.config.exclude.length; num++){
-        if (this.exclude.indexOf(this.config.exclude[num]) == -1){
-          this.exclude.push(this.config.exclude[num]);
-        }
-      }
-    }
-    if (this.config.lost){
-      this.zones.push(this.config.lost);
-    }
     if (this.config.travelling){
       this.zones.push(this.config.travelling);
     }
+    if (this.config.lost){
+      this.zones.push(this.lostState);
+    }
 
     for (num = 0; num < this.config.wizards.length; num++){
       if (!this._hass.states[this.config.wizards[num].entity])
         throw new Error("Unable to find state for entity " + this.config.wizards[num].entity);
```

Comment:
Instead of throwing an error, consider logging a warning and skipping that entity. This would prevent the entire card from breaking if one entity is missing.